src/js/pages/Process/TwitterSignInProcess.jsx

```diff
@@ -58,7 +59,7 @@ export default class TwitterSignInProcess extends Component {
         pathname: '/settings/account',  // SnackNotifier that handles this is in SettingsDashboard
         state: {
           message: 'Twitter sign in failed. Please try again.',
-          message_type: 'success',
+          message_type: 'warning',
```

Comment:
May want to be 'severity' instead of 'message_type'

Reply:
Yes